src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/policy/AdditionalConditionPss.kt

```diff
@@ -8,6 +8,6 @@ data class AdditionalConditionPss(
   override val requiresInput: Boolean,
   override val categoryShort: String? = null,
   val pssDates: Boolean? = null,
-  override val inputs: List<PssInput>? = null,
```

Comment:
Remove the generic nature of these lists of inputs and removed the concept of PSSInputs.

This merely removed the ability to have the same level of rich formatting as Input which is a bit of artificial restriction